src/__tests__/Details.test.js

```diff
@@ -0,0 +1,16 @@
+import { BrowserRouter as Router } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import renderer from 'react-test-renderer';
+import store from '../Redux/configureStore';
+import Details from '../components/Details';
+
+it('testing rendering Details component', () => {
+  const tree = renderer.create(
+    <Provider store={store}>
+      <Router>
+        <Details />
+      </Router>
+    </Provider>,
+  ).toJSON;
+  expect(tree).toMatchSnapshot();
+});
```

Comment:
- [ ] <b>REQUIRED:</b> You only tested Snapshot. As the project requires, kindly add some unit and integration tests as well.
![Screen Shot 2022-03-13 at 04 35 40](https://user-images.githubusercontent.com/45217364/158042544-798fcf2a-319a-4b20-b0d5-83d5acb38676.png)